pcsx2/Recording/InputRecording.cpp

```diff
@@ -332,13 +333,13 @@ bool InputRecording::Play(wxString fileName)
 		if (!CoreThread.IsOpen())
 		{
 			recordingConLog(L"[REC]: Game is not open, aborting playing input recording which starts on a save-state.\n");
-			inputRecordingData.Close();
+			Stop();
```

Comment:
These extra Stop() calls don't add anything extra. By this point, Stop() would've already been called meaning state is already set to NotActive, incrementUndo is set to false, and the previous file is closed. However, OpenExisting() has loaded another file.
If after that, the loading of the savestate fails, the only thing that has been changed is the loaded file. Therefore, I only called Close() as it would be redundant otherwise. Additionally, it would add an extra log entry that isn't necessary.

Reply:
This is true, though other than the logging concern, I'm not concerned with redundancy.  I'm more concerned with simplifying and centralizing logic -- ie. `Stop()` handles everything that needs to be handled for cleaning up the recording process.  I'll undo this for the logging reason for now, but I think a better solution might to to allow a message to be passed in to the `Stop` method that is logged instead of the default one.